### PR TITLE
Class 7

### PR DESCRIPTION
**Pull request by patrickjcolgan**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+  render: function() {
+    this.generateHourly();
+    var sectionEl = document.getElementById("pikePlaceStore");
+    sectionEl.textContent = this.name;
+    var ulEl = document.createElement("ul");//creates ul
+
+    //creates li for each hourOpen
+    for (hour in hoursOpen) {
+      var liEl = document.createElement("li");
+      liEl.textContent = hoursOpen[hour] + this.hourlySales[hour];//each hour + totalSales
```
Comment (katepaulino) [inline app.js:30]: add a colon so the hours and the hourly sales have a space

File: app.js [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+  totalSales: 0,
+  randomCustomer: function () {
+    return Math.floor(Math.random() * (this.max - this.min) + this.min);
+  },
+  //use for loop to generate a random number and multiply it by the average and stick in hourly sales. add to total sales
+  generateHourly: function () {
+    for (var i = 0; i < hoursOpen.length; i++) {
+      this.hourlySales.push(Math.floor(this.avgSales * this.randomCustomer(this.min, this.max)));
+      this.totalSales = this.totalSales + this.hourlySales[i];
+    }
```
Comment (katepaulino) [inline app.js:197]: your generate hourly function is good but can be refactored.

File: index.html [hunk 3]
```diff
@@ -1 +1,27 @@
...
+      </section>
+      <section>
+        <p id="southCenterStore"></p>
+      </section>
+      <section>
+        <p id="bellevueSquareStore"></p>
+      </section>
+      <section>
+        <p id="alkiStore"></p>
+      </section>
```
Comment (katepaulino) [inline index.html:23]: you can remove your p tags and move your id's to the section tags.

Comment (patrickjcolgan) [inline index.html:23]: Thanks. Wasn't sure before, but after today's code review, it's obvious why I don't need the.

File: app.js (hunk 2 above)
Comment (patrickjcolgan) [inline app.js:197]: Do you mean like how Scott refactored Jonathan's during code review?

Comment (katepaulino) [inline app.js:197]: You can use the += on line 17 (D.R.Y)

**Pull request merged**